datalad/distribution/add.py

```diff
@@ -119,8 +119,16 @@ class Add(Interface):
             transport"""),
         recursive=recursion_flag,
         recursion_limit=recursion_limit,
+        ds2super=Parameter(
+            args=("--ds2super", "--datasets-to-super",),
```

Comment:
`-S` is only used in `save` as a shortcut to `--super-datasets`. Will do it then...